### DISCUSSION TITLE
No GFX configuration option

### DISCUSSION THREAD

**Discussion by sujit9511** [DavidS95/Smokeless_UMAF#42] [Q&A] [answered]
Ref : https://youtu.be/0jwrWCF5fhc?si=QAooljKXRP-95Hwr

My system : Ryzen 5 2500u Acer aspire 3 a315-41 laptop. 

I tried increasing VRAM like shown in that video. But I do not see GFX config option. Need help.

**Comment by ForeverDeux** [comment, marked as answer]
The video says it does not work on the 2000 series.
![image](https://github.com/DavidS95/Smokeless_UMAF/assets/145818345/8510bf58-7b9f-4344-bf15-7d403ca757e5)

**Reply by J0r2** [reply]
Is there any workaround to enable those settings on a Ryzen 2000?

**Reply by sujit9511** [reply]
Nope, but I upgraded my ram to 16 gb and changed HDD to SSD. My laptop gave me much better performance
